addons/udes_stock/models/stock_warehouse.py

```diff
@@ -67,3 +67,22 @@ def get_picking_types(self):
             raise ValidationError(_("Cannot find picking types for warehouse %s.") % self.name)
 
         return picking_types
+    
+    def _get_sequence_values(self):
+        """
+        Purpose: Overwrite the hardcoded sequence values. This will mean that when the warehouse is created the sequences for each picking type
+        will not be created with the old Odoo prefix (The sequences should not be created at all). When the warehouse name is updated it means
+        the sequences for the picking types will not be updated with the old Odoo prefix. 
+        """
+        return {
```

Comment:
Is black really formatting the empty dict lines like this? :vomiting_face:

Reply:
How would you format it out of interest?

(Not for the story as black is king, but just curious!)

Reply:
It turns out I'd format it the same as black would ;)

```diff
black --diff test.py 
--- test.py	2022-05-17 11:21:23.511293 +0000
+++ test.py	2022-05-17 12:48:25.233572 +0000
@@ -1,11 +1,7 @@
 def foo():
     return {
-        'aaaa_type_id': {
-        },
-        'bbbb_type_id': {
-        },
-        'cccc_type_id': {
-        },
-        'dddd_type_id': {
-        },
+        "aaaa_type_id": {},
+        "bbbb_type_id": {},
+        "cccc_type_id": {},
+        "dddd_type_id": {},
     }
would reformat test.py
```